contracts/optimistic-ethereum/libraries/utils/Lib_RingBuffer.sol

```diff
@@ -17,12 +13,11 @@ library Lib_RingBuffer {
     }
 
     struct RingBuffer {
-        bytes32 id;
```

Comment:
We no longer need an ID, was only used in tandem with `overwriter`, which is removed by making this push based (see `nextOverwritableIndex` below).